bindata/assets/kueue-operator/networkpolicy/allow-ingress-visi

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Did the filename of this get truncated?  Looks like it wanted to be allow-ingress-visibility.yaml based on how the other files are named.

Reply:
yes, not sure how it happened, let me fix it, good catch :)